python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -1119,41 +1127,23 @@ def test_torch_tensor_nccl_all_reduce_custom_comm_wrong_actors(ray_start_regular
     """
     Test an error is thrown when an all-reduce binds to a wrong set of actors.
     """
-    if not USE_GPU:
-        pytest.skip("NCCL tests require GPUs")
-
-    assert (
-        sum(node["Resources"].get("GPU", 0) for node in ray.nodes()) > 1
-    ), "This test requires at least 2 GPUs"
-
-    actor_cls = TorchTensorWorker.options(num_cpus=0, num_gpus=1)
+    actor_cls = TorchTensorWorker.options(num_gpus=0)
 
     num_workers = 2
     workers = [actor_cls.remote() for _ in range(num_workers)]
 
-    class TestNcclGroup(GPUCommunicator):
+    class DummyNcclGroup(GPUCommunicator):
```

Comment:
Use `MockNcclGroup`.

Reply:
This change is a response to Stephanie's comment about reducing the number of tests that require GPUs. Because `MockNcclGroup` uses GPU, switching to `MockNcclGroup` requires this test to use GPUs.